dashboard/src/components/DeploymentForm/DeploymentForm.tsx

```diff
@@ -201,6 +190,28 @@ class DeploymentForm extends React.Component<IDeploymentFormProps, IDeploymentFo
   public handleValuesChange = (value: string) => {
     this.setState({ appValues: value, valuesModified: true });
   };
+
+  private renderBasicForm = () => {
```

Comment:
I'd keep both as separate components instead of methods inside the same component